karma.config.js

```diff
@@ -0,0 +1,70 @@
+var argv = require('yargs').argv;
```

Comment:
Hmm, I'm on node `0.12.10` and npm `2.14.9` and on clean `npm install` of this branch a subsequent `npm run test` throws this error (assuming the above config error is fixed). 
```
03 03 2016 15:19:39.578:ERROR [config]: Invalid config file!
  Error: Cannot find module 'yargs'
```